contents/teams/replay/objectives.mdx

```diff
@@ -1,17 +1,24 @@
-### 🎯 Goal 1: 10 Happy Android Replay Users 
-
-We'll also release an alpha of iOS Mobile Replay.
+### 🎯 Goal 1: Quality and Consolidation
 
 Why? 
 
-We've done the work's base and now we want to release the new Android SDK with session replay and onboard customers to validate the product. 
+We've done a lot of investigation and preparation work and we want to capitalize on that
+
+* Mobile replay out of beta for Android/iOS
+* Build the best UX for capturing and playing recordings
+* Build the best UX for finding recordings
+
+### 🎯 Goal 2: Error tracking MVP
 
-### 🎯 Goal 2: People love replay even more
+Why? PostHog is "The single platform to analyze, test, observe, and deploy new features". Having network timings, network payloads, and console errors in Session Replay is a great developer tool. We want to learn by doing to understand how we could capture and expose even more information here to help people dig into what is happening with their products
 
-Why? - we want this product to be fit for everyone and understand where it doesn't spark joy and how we can increase the value people receive and feel.
+* Building on top of the existing `$exception` ingestion
+* In the hands of real users getting feedback on solving their problems
```

Comment:
```suggestion
* Getting it in the hands of real users, so that we can collect feedback on how to best solve their problems
```

-> the wording makes more sense to me this way